src/ui/view/share/data/tagsField/tagsField.tsx

```diff
@@ -0,0 +1,71 @@
+import { useCallback, type FC, useState } from 'react'
+import classNames from 'classnames'
+import { Trans } from 'react-i18next'
+import { Tag } from '@/ui/component/tag/tag'
+import '../i18n/index'
+import './tagsField.scss'
+
+type TagsFieldProps = {
+  tags: string[]
+  addTag: (tag: string) => void
+  removeTag: (tag: string) => void
+}
+
+// eslint-disable-next-line max-lines-per-function
+export const TagsField: FC<TagsFieldProps> = ({ tags, addTag, removeTag }) => {
+  const [value, setValue] = useState('')
+
+  const handleChange = useCallback(
+    (event: React.ChangeEvent<HTMLInputElement>) => {
+      setValue(event.target.value)
+    },
+    [setValue]
+  )
+
+  const handleKeyDown = useCallback(
+    (event: React.KeyboardEvent<HTMLInputElement>) => {
+      if (event.key === 'Enter') {
```

Comment:
This might not work in some older browsers we also need to handle `event.keyCode === 13`

Reply:
Which browser are you thinking of? I think the support is good for the required browsers. https://caniuse.com/?search=event.key

Reply:
Im used to handling key codes and key to make sure everything works everywhere :)